espnet2/tts/gst/style_encoder.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
```

Comment:
`coding: utf-8` has no effect if python3.
Pep8 recommends not writing the statement now.